src/style.css

```diff
@@ -1 +1,90 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 600;
+  font-style: normal;
+}
+
+html {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
The font-family 'Avenir' is used here, but it is not defined in the @font-face rules. Ensure that 'Avenir' is available in your environment or consider using a fallback font.